### PR TITLE
cbindgen, add new version

### PR DESCRIPTION
**Pull request by Begasus**

### CODE REVIEW

**Comment by Begasus** [comment]
Builds and packages fine on both arch's.

File: dev-rust/cbindgen/cbindgen-0.26.0.recipe [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+	cmd:cargo$secondaryArchSuffix >= 0.70
+	cmd:gcc$secondaryArchSuffix
+	"
+
+defineDebugInfoPackage cbindgen$secondaryArchSuffix \
+	"$commandBinDir"/cbindgen
+
+BUILD()
+{
+	cargo fetch --locked
```
Comment (korli) [inline dev-rust/cbindgen/cbindgen-0.26.0.recipe:41]: shouldn't it specify --target ? instead of all architectures.

Comment (korli) [reply dev-rust/cbindgen/cbindgen-0.26.0.recipe:41]: Arch uses: ``cargo fetch --locked --target "$(rustc -vV | sed -n 's/host: //p')"`` . Don't know if it works on Haiku.

Comment (kenz-gelsoft) [reply dev-rust/cbindgen/cbindgen-0.26.0.recipe:41]: korli's suggested command seems to work
```
~/src/gecko-dev> rustc -vV | sed -n 's/host: //p'
x86_64-unknown-haiku
```

Comment (Begasus) [reply dev-rust/cbindgen/cbindgen-0.26.0.recipe:41]: Will run some checks on this here, got/am side-tracked again. :)

Comment (Begasus) [reply dev-rust/cbindgen/cbindgen-0.26.0.recipe]: Change applied, haven't checked 32bit yet (won't be for today).

**Comment by Begasus** [comment]
@kenz-gelsoft I see upstream has a new version, but iirc you needed this specific version?

**Comment by kenz-gelsoft** [comment]
> @kenz-gelsoft I see upstream has a new version, but iirc you needed this specific version?

Yes. 0.27 won't work with current firefox port. We can rebase the port to upstream version to fix this, but it takes some time. If you never mind, I want 0.26 for now.

**Pull request merged**